db/migrate/20140811094300_rename_deferred_columns_on_post_action.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class RenameDeferredColumnsOnPostAction < ActiveRecord::Migration[4.2]
+  def change
+    rename_column :post_actions, :deferred_by_id, :deferred_by_id
+    rename_column :post_actions, :deferred_at, :deferred_at
```

Comment:
obviously this change should be dropped. Although I do wonder about the file rename. I'll probably drop it as well.